cmp/grid/GridAutosizeMode.js

```diff
@@ -0,0 +1,35 @@
+/*
```

Comment:
Should we move this to `core/enums`?

Reply:
I don't think they all have to live there.  Those ones are there because they are shared